tsinfer/inference.py

```diff
@@ -2179,3 +2298,93 @@ def minimise(ts):
         filter_individuals=False,
         filter_populations=False,
     )
+
+
+def solve_num_mismatches(ts, k):
```

Comment:
I'm not quite following here, but I think that calculating the low-level recombination probability depends on the distance between the two sites, and I can't see that being used here. That might be OK for your usage, however.